src/Microsoft.ML.Legacy/CSharpApi.cs

```diff
@@ -400,6 +400,20 @@ public void Add(Microsoft.ML.Legacy.Models.PAVCalibrator input, Microsoft.ML.Leg
                 _jsonNodes.Add(Serialize("Models.PAVCalibrator", input, output));
             }
 
+            [Obsolete]
+            public Microsoft.ML.Legacy.Models.PipelineSweeper.Output Add(Microsoft.ML.Legacy.Models.PipelineSweeper input)
+            {
+                var output = new Microsoft.ML.Legacy.Models.PipelineSweeper.Output();
+                Add(input, output);
+                return output;
+            }
+
+            [Obsolete]
+            public void Add(Microsoft.ML.Legacy.Models.PipelineSweeper input, Microsoft.ML.Legacy.Models.PipelineSweeper.Output output)
+            {
+                _jsonNodes.Add(Serialize("Models.PipelineSweeper", input, output));
+            }
```

Comment:
Nope, this is gone. 

Delete everything inside your bin folder, and re-build from root before regenerating the API.